.github/workflows/checkpoint.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+name: Checkpoint UDS Core
+
+on:
+  pull_request:
+    # milestoned is added here as a workaround for release-please not triggering PR workflows (PRs should be added to a milestone to trigger the workflow).
+    types: [milestoned, opened, reopened, synchronize]
+  # triggered by tag-and-release.yaml
+  workflow_call:
+
+jobs:
+  publish-uds-core:
+    strategy:
+      matrix:
+        architecture: [amd64, arm64]
+    runs-on: ${{ matrix.architecture == 'arm64' && 'uds-ubuntu-arm64-4-core' || 'uds-ubuntu-big-boy-4-core' }}
+    name: Publish checkpoint
```

Comment:
Similar name comment here if this stays as a test job as well (and not always publish)